Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM python:3.7
+
+MAINTAINER support@cognite.com
+
+ENV PYTHONFAULTHANDLER=1 \
+  PYTHONUNBUFFERED=1 \
+  PYTHONHASHSEED=random \
+  PIP_NO_CACHE_DIR=off \
+  PIP_DISABLE_PIP_VERSION_CHECK=on \
+  PIP_DEFAULT_TIMEOUT=100 \
+  POETRY_VERSION=0.12.16
+
+# System deps:
+RUN pip install "poetry==$POETRY_VERSION"
+
+# Copy only requirements to cache them in docker layer
+WORKDIR /code
+COPY poetry.lock pyproject.toml /code/
+
+# Creating folders, and files for a project:
+COPY . /code
+
+# Project initialization:
+RUN poetry config settings.virtualenvs.create false && poetry install --no-dev --no-interaction --no-ansi
+
+ENTRYPOINT ["python", "-m", "cognite.replicator"]
+CMD []
```

Comment:
I'm not sure if we need this line.

Reply:
I'm assuming that CMD is inherited from the parent dockerfile [1] (not sure if it is true).
See [2] for discussion on ENTRYPOINT vs CMD. I think this setup is close to best practice.

[1] https://github.com/docker-library/python/blob/c3233a936f58bee7c6899d3e381f23ed12cfc7a8/3.7/buster/Dockerfile
[2] https://aws.amazon.com/blogs/opensource/demystifying-entrypoint-cmd-docker/

Reply:
I tested the assumption, and it was not needed. Removed the CMD. Good suggestion! :)